lib/api/storage.ts

```diff
@@ -18,11 +20,15 @@ export class ShredderStorage {
   }
 
   static getRaw() {
-    return localStorage.getItem(`${ location.origin }-shredder`);
+    return localStorage.getItem(this.storageKey);
   }
 
   static getSerialized() {
```

Comment:
localStorage is shared between all hosts, so we have to make a separate key for each website. That was the reason to use location.origin as a part of the key.